lib/generators/doorkeeper/templates/migration.rb

```diff
@@ -39,13 +39,14 @@ def change
       # https://github.com/doorkeeper-gem/doorkeeper/tree/v3.0.0.rc1#custom-access-token-generator
       #
       # t.text     :token,             null: false
-      t.string   :token,             null: false
+      t.string   :token,                  null: false
 
       t.string   :refresh_token
       t.integer  :expires_in
       t.datetime :revoked_at
-      t.datetime :created_at,        null: false
+      t.datetime :created_at,             null: false
       t.string   :scopes
+      t.string   :previous_refresh_token, null: false, default: ""
```

Comment:
Or is the `refresh_token_revoked_on_use` configuration still a thing? If so, we should add it with a comment to the initializer template.